<launch>
    <node pkg="orb_slam_3_ros" type="orb_mono" name="orb_mono" args="$(find orb_slam_3_ros)/vocabulary/ORBvoc.txt $(find app_urssa)/config/UnityCam.yaml" />
</launch>